gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -202,10 +206,6 @@ gst_tensor_query_serversink_change_state (GstElement * element,
     case GST_STATE_CHANGE_PLAYING_TO_PAUSED:
       gst_tensor_query_server_release_edge_handle (sink->sink_id);
       break;
-    case GST_STATE_CHANGE_PAUSED_TO_PLAYING:
-      caps = gst_pad_peer_query_caps (GST_BASE_SINK_PAD (bsink), NULL);
-      gst_tensor_query_serversink_set_caps(bsink, caps);
-      break;
     default:
       break;
```

Comment:
It seems like now the switch statement has only one case. maybe it can use an if statement?

```
if (transition == GST_STATE_CHANGE_PLAYING_TO_PAUSED) {
   gst_tensor_query_server_release_edge_handle (sink->sink_id);
}
```

Reply:
possible, however for future work, use switch-case (guide from gst sample, I am not sure but if adding another state change is required later, better remaining current code.)